src/client/components/ShoppingCart/ShoppingCart.styles.css

```diff
@@ -0,0 +1,118 @@
+.cart-container {
+  border: 2px solid rgb(187, 183, 183);
+  width: 650px;
+  /* height: 350px; */
+
+  margin: 0;
+  padding: 0;
+}
+
+.cart-product-container {
+  display: flex;
+  flex-direction: row;
+  width: 646px;
+  padding: 30px;
+  background-color: #fff;
+  justify-content: space-around;
+}
+
+.image-discount-box {
+  position: relative;
+}
+.add_cart_button {
+  margin-top: auto;
+  padding: 5px;
+  align-items: flex-end;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+  height: 100px;
+  padding-left: 100px;
+}
+
+.discount {
+  position: absolute;
+  top: 0;
+  right: 0;
+  width: 38px;
+  height: 32px;
+  background-color: #fff;
+  opacity: 0.8;
+  border: 1px solid #9b8d8d;
+  font-size: 1.2rem;
+  font-weight: bolder;
+  color: black;
+}
+
+.discounted_price {
+  display: flex;
```

Comment:
Make use of special offer products and can avoid dual code.

Reply:
done